build/craft-netlify-redirects.js

```diff
@@ -26,6 +26,14 @@ export default function() {
 		// Open up _redirects
 		const file = resolve(this.nuxt.options.srcDir, 'dist/_redirects')
 		let redirects = existsSync(file) ? readFileSync(file, 'utf8') : ''
+		
+		let data = { query: getEntries }
+
+		// if there's a CMS_SITE env var, then add it
+		// I tested this in craft and it didn't like an empty string
+		if (process.env.CMS_SITE) {
+			data.site = process.env.CMS_SITE
+		}
```

Comment:
Wouldn't this need to be like `data.variables = { site: process.env.CMS_SITE }`?  

Also, check out this alternative:

```js
		const response = await axios({
			url: process.env.CMS_ENDPOINT,
			method: 'post',
			headers: {
				'Content-Type': 'application/json'
			},
			data: {
				query: getEntries,
				variables: {
					site: process.env.CMS_SITE || undefined
				}
			}
		});
```

Like could you test if converting emptry string to an `undefined` would be treated by Craft GQL API like it wasn't present?

Reply:
I tried `undefined` and it didn't like that. Though it did seem to be ok with `null`

Reply:
@weotch I tried to `yarn link` my local cloak to P&B so I could run a `yarn generate` to see if this worked, but I was running into dependency issues when linked

Reply:
I pushed up that edit in the mean time

Reply:
Yeah, i ran into dependency issues too.  Trying an SSG now with this in package.json 🤞 

![image](https://user-images.githubusercontent.com/77567/140261877-086cf9eb-eae6-4021-84d4-a09dfd8beff0.png)

Reply:
It worked after fixing d55893d